lib/readyset.rb

```diff
@@ -80,13 +81,16 @@ def self.raw_query(*sql_array) # :nodoc:
   # @yield a block whose queries should be routed to ReadySet.
   # @return the value of the last line of the block.
   def self.route(prevent_writes: true, &block)
+    QueryAnnotator.routing_to_readyset = true
     if prevent_writes
       ActiveRecord::Base.connected_to(role: reading_role, shard: shard, prevent_writes: true,
         &block)
     else
       ActiveRecord::Base.connected_to(role: writing_role, shard: shard, prevent_writes: false,
         &block)
     end
+  ensure
```

Comment:
Arghhh I realized that this approach won't quite work. It's possible that people may invoke `ActiveRecord::Base.connected_to` within a `Readyset.route` block, which would make us think we were routing to ReadySet when we actually weren't. That said, I discovered an even simpler approach. We can add a query log tag like so:
```ruby
Rails.configuration.active_record.query_log_tags << {
  destination: ->(context) do
    ActiveRecord::Base.connection_db_config.name
  end,
end
```
This will include a `destination` tag that reads either `primary` or `readyset` depending on where the query was headed.